tests/standalone-vault/data.tf

```diff
@@ -0,0 +1,3 @@
+data "aws_secretsmanager_secret" "tfe_license" {
+  name = module.secrets.tfe_license
+}
```

Comment:
We shouldn't need this data source if we update the root module inputs to align with the secrets fixture outputs.

Reply:
removed